docs/UserGuide.md

```diff
@@ -269,24 +269,25 @@ Adds a lesson with the corresponding details to the specified student in TAB.
 **Lesson fields:**
 * Date of lesson `dd MMM yyyy`
   *  e.g. `02 Jan 2018`
-* Start time `HH:mm`
-* End time `HH:mm`
+* Time range `HHmm-HHmm`
 * Subject
 * Homework
 
-Format: `ladd INDEX [recurring/] date/dd MMM yyyy start/HH:mm end/HH:mm subject/SUBJECT [hw/HOMEWORK]`
+Format: `ladd INDEX [recurring/] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT [hw/HOMEWORK]…​`
+
+* The date is case-insensitive. i.e. `12 jaN 2022` is equivalent to `12 JAN 2022`.
```

Comment:
i think you can have a "Notes about lesson add command" block and merge with the tip below.
maybe lesson delete also can put inside a block